secure_message/repository/retriever.py

```diff
@@ -242,6 +248,18 @@ def _retrieve_thread_for_internal_user(thread_id):
 
         return result
 
+    @staticmethod
+    def retrieve_thread_metadata(thread_id):
+        result = Conversation.query.filter(Conversation.thread_id == thread_id)
+        try:
+            return result.one()
+        except NoResultFound:
+            logger.error("No conversation found", thread_id=thread_id)
+            raise NotFound(description=f"Conversation with thread_id '{thread_id}' does not exist")
```

Comment:
A conversation record won't always exist when we first deploy these changes (we'll need to create conversation records for all existing conversations after we've deployed this), so I don't think we should class no conversation record as an error.

Reply:
Agreed, I thought the same thing and thought I'd changed it to that.  Must've changed it somewhere else sorry >_<